experimental/dds/tree2/src/shared-tree-core/branch.ts

```diff
@@ -186,6 +191,7 @@ export class SharedTreeBranch<TEditor extends ChangeFamilyEditor, TChange> exten
 		// If this is not part of a transaction, add it to the undo commit tree
 		if (undoRedoType !== undefined && !this.isTransacting()) {
 			this.undoRedoManager?.trackCommit(this.head, undoRedoType);
+			this.emit("undoable", undoRedoType);
```

Comment:
This is emitted for all operations that call `applyChange` and also for transaction commits. What about for rebases and merges? Don't those update the undo/redo stack?